src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,101 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd: (newMovie: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const emptyState = {
+    count: 0,
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+    title: '',
+    description: '',
+  };
```

Comment:
Can you move it outside the component? It's. better, because React will not create a new constant each render in this case
```suggestion

const DEFAULT_STATE = { ...

export const NewMovie: React.FC<Props> = ({ onAdd }) => {
```